go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/grpc-ecosystem/grpc-gateway v1.14.3
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/lndclient v0.11.0-3
+	github.com/lightninglabs/lndclient v1.0.1-0.20210111133012-a2e3bbe2d814
```

Comment:
I pushed a tag `v0.11.0-5`, please use that.

Reply:
done!